namnder/systemgruppen/sidebar_sv.md

```diff
@@ -10,12 +10,13 @@ Mathias Magnusson (förra)</br>
 
 [ior@datasektionen.se](mailto:ior@datasektionen.se)
 
-#### IORs Mattermost
+#### Systemgruppens Mattermost
 
-<https://mattermost.datasektionen.se/ior/channels/town-square>
+<https://mattermost.datasektionen.se/systemgruppen/channels/town-square>
```

Comment:
```suggestion
<https://dsekt.se/ior>
```

Reply:
Oh yeah I need to update the dsekt link to systemgruppen too. Rename the Mattermost first so I know for sure what to link to.